src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -150,7 +153,34 @@ public PersistentDataStore() {
 
     return messages;
   }
-
+  
+  public List<Profile> loadProfile() throws PersistentDataStoreException {
+
+	    List<Profile> profiles = new ArrayList<>();
+
+	    // Retrieve all messages from the datastore.
+	    Query query = new Query("chat-profile").addSort("creation_time", SortDirection.ASCENDING);
+	    PreparedQuery results = datastore.prepare(query);
+
+	    for (Entity entity : results.asIterable()) {
+	      try {
+	        UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
+	        UUID profileUuid = UUID.fromString((String) entity.getProperty("profile_uuid"));
```

Comment:
Same here for renaming the properties (`id` instead of `profile_uuid`, `owner_id` instead of `uuid`)